drivers/syslog/vsyslog.c

```diff
@@ -206,7 +206,10 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
   /* Prepend the process name */
 
   tcb = nxsched_get_tcb(getpid());
-  ret += lib_sprintf(&stream.public, "%s: ", tcb->name);
+  if (tcb != NULL)
+    {
+      ret += lib_sprintf(&stream.public, "%s: ", tcb->name);
```

Comment:
To not hide the fact. What about `tcb == NULL ? "(null)" : tcb->name` in the sprintf?

Reply:
Nice suggestion! I've implemented it, but I took the liberty of testing `tcb != NULL`, which is the most common case.